source/includes/_lips_site_status.md

```diff
@@ -0,0 +1,30 @@
+## Site status
+<a name="lips_site_status"></a>
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /site_status/[gmaid]
+
+Use GET to retrieve site status. Data is returned for a GMAID. Data is an object of call, form and chat first and last events in date time format and also a flag "active" for the status of the site.
```

Comment:
You can remove this sentence entirely
```
Data is an object of call, form and chat first and last events in date time format and also a flag "active" for the status of the site.
```

It's not accurate anymore and this API is pretty easy to understand.